var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -32,8 +32,10 @@ class Relion(CMakePackage):
     electron cryo-microscopy (cryo-EM)."""
 
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
-    url      = "https://github.com/3dem/relion/archive/2.0.3.tar.gz"
```

Comment:
you sure `2.0.3` is still working with this change?

Reply:
It does but it's no longer recommended to be used so I removed it as a release option

Reply:
what I meant is that in the previous version, when `2.0.3` was around, I don't think Spack would be able to guess url to fetch it.

Reply:
I am going to add 2.0.3 back now.  2.1 is actually beta.